src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt

```diff
@@ -0,0 +1,43 @@
+package com.hibob.academy.employeeFeedback.service
+
+import com.hibob.academy.employeeFeedback.dao.*
+import jakarta.ws.rs.BadRequestException
+import org.springframework.stereotype.Service
+
+@Service
+class FeedbackService(private val feedbackDao: FeedbackDao, private val employeeDao: EmployeeDao) {
+    fun submitFeedback(employeeId: Int, companyId: Int, anonymity: AnonymityType, feedback: String): Boolean {
+        var employeeIdForFeedback: Int? = employeeId
+        if (anonymity == AnonymityType.ANONYMOUS)
+            employeeIdForFeedback = null
+
+        if (feedback.length < 30 || feedback.length > 500)
```

Comment:
One of the rules in clean code is that a function should do only one thing and this should be clear.
In this case you are doing 2 things - validating and submitting. 
Of course you need to validate before submitting, but think of how this function will look if you need to add another type of validation?

Reply:
Also - the numbers in the validation, it's better if they are consts. Specifically if you use them in multiple places

Reply:
noted. I'll fix it